app/rev2/prelaunch/mock/ignition.c

```
@@ -27,4 +27,11 @@ bool ign_main_cont(void) {
     } else {
         return false;
     }
- }
\ No newline at end of file
+ }
+ IGN_STATUS ign_cmd_execute
```

Cppcheck (2.20.0)

[style] The function 'ign_cmd_execute' is never used.

(unusedFunction)